drivers/at/at.c

```diff
@@ -275,53 +312,66 @@ ssize_t at_send_cmd_get_resp_wait_ok(at_dev_t *dev, const char *command, const c
 {
     ssize_t res;
     ssize_t res_ok;
-    char ok_buf[64];
+    char *err_buff = dev->err_buf;
+    size_t err_buff_size = dev->err_buf_size;
```

Comment:
Why do we need a local variable? If it is for readability, then it does not help much. (In fact you present the reader two extra lines to think about.) If it is to optimize, then don't do that. The compiler is more clever than you may think.

And why a different spelling?